spring-kafka-test/src/main/java/org/springframework/kafka/test/utils/KafkaTestUtils.java

```diff
@@ -239,6 +242,38 @@ public static OffsetAndMetadata getCurrentOffset(String brokerAddresses, String
 		}
 	}
 
+	/**
+	 * Return the end offsets of the requested topic/partitions
+	 * @param <K> the key type.
+	 * @param <V> the value type.
+	 * @param consumer the consumer.
+	 * @param topic the topic.
+	 * @param partitions the partitions, or null for all partitions.
+	 * @return the map of end offsets.
+	 * @since 2.6.5
```

Comment:
We probably need to add a `@see` for JavaDocs of that `consumer.endOffsets()` because that is exactly an info we get back.